deployment/manifests/default.pp

```diff
@@ -31,6 +31,14 @@
   require     => Exec['create_guest'],
 }
 
+# Django-docker-engine needs a place for ephemeral data.
+# In production, this is a separate EBS mount, so we don't need to create it locally.
+file { '/data':
+  ensure => 'directory',
+  owner  => $app_user,
+  group  => $app_group
+}
+
```

Comment:
This should be combined into a separate module with storage related items in [aws.pp](https://github.com/refinery-platform/refinery-platform/blob/mccalluc/django-docker-data-dir/deployment/refinery-modules/refinery/manifests/aws.pp).

Reply:
@hackdna Are you saying create a `storage.pp`? 
Would this be better suited for an issue outside of this pr?